src/firebase.js

```diff
@@ -4,14 +4,13 @@ import { getMessaging, getToken, onMessage, isSupported } from 'firebase/messagi
 import * as config from './config';
 
 
-let firebase = null;
 let messaging = null;
 let supported = false;
 
 const initializeFirebase = async () => {
 	supported = await isSupported();
 	if (supported) {
-		firebase = initializeApp(config.FIREBASE);
+		initializeApp(config.FIREBASE);
```

Comment:
Ah, I wasn't sure if this would work, or if this might make the app instance get garbage-collected. Nice!